docs/csharp/language-reference/keywords/where-generic-type-constraint.md

```diff
@@ -10,57 +10,60 @@ f1_keywords:
   - "whereconstraint_CSharpKeyword"
 helpviewer_keywords: 
   - "where (generic type constraint) [C#]"
-ms.assetid: d7aa871b-0714-416a-bab2-96f87ada4310
-caps.latest.revision: 10
 author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # where (generic type constraint) (C# Reference)
-In a generic type definition, the `where` clause is used to specify constraints on the types that can be used as arguments for a type parameter defined in a generic declaration. For example, you can declare a generic class, `MyGenericClass`, such that the type parameter `T` implements the <xref:System.IComparable%601> interface:  
-  
-```csharp  
-public class MyGenericClass<T> where T:IComparable { }  
-```  
-  
+
+The `where` clause in a generic definition specifies constraints on the types that are used as arguments for type parameters in a generic type, method, delegate, or local function. For example, you can declare a generic class, `MyGenericClass`, such that the type parameter `T` implements the <xref:System.IComparable%601> interface:
```

Comment:
I think that I'd add a summary sentence after the first sentence. Something like, "Constraints can specify interfaces, base classes, classes. structs or unmanaged types..." 
Then I'd also make the interface constraint a separate paragraph, since the interface constraint is one of the types of constraints. Here it just appears to be an example of some kind of constraint.